### ISSUE TITLE
Typing too fast in CAPTCHA results in letters not being produced in the the box

### ISSUE THREAD

**Issue by reportthings** [K1rakishou/Kuroba-Experimental#694]
If the captcha is "AJABA" (just an example) I can quickly type AJABA. But if I type any faster than very slow, it will fill in as "AA" and skip all letters in between.

**Comment by K1rakishou** [comment]
Yeah I remember someone saying about this weird behavior but I can't reproduce it myself. The captcha UI is built with the Jetpack Compose™ so it might be a weird bug that only affects specific devices or even only keyboards.
By the way, by "too fast" how fast exactly do you mean? Can you show a video?

**Comment by reportthings** [comment]
Here is a video showcasing it. I spammed keys at about the same speed in the message box and captcha box. 

https://user-images.githubusercontent.com/85461551/147766860-eadb892f-2c12-43c3-97dd-0c5060076422.mp4

**Comment by reportthings** [comment]
The issue has only been present since late November or early December. Before that I never experienced it.  I tried to scour through and see if I could find which update it was but I honestly can't definitively say.

**Comment by K1rakishou** [comment]
Yeah I think I can reproduce it now, but to me I need to type way faster than I usually type for it to start skipping keys. I will try to look what's causing this. Weird thing is when I'm using an emulator and typing on a PC keyboard it works just fine, only skips buttons when using phone keyboard on an actual phone.

**Comment by reportthings** [comment]
It's not a huge problem for me. I mostly notice it when the captcha begins with letters typed by each hand in succession, such as "AL" where it will skip the L.   Thanks for looking into it
